src/Components/ServiceScene/LineFilterScene.tsx

```diff
@@ -35,8 +35,8 @@ export class LineFilterScene extends SceneObjectBase<LineFilterState> {
   };
 
   private getVariable() {
-    const variable = sceneGraph.lookupVariable(VAR_LINE_FILTER, this);
-    if (!(variable instanceof CustomVariable)) {
+    const variable = getLineFilterVariable(this);
+    if (!variable) {
       throw new Error('Logs format variable not found');
```

Comment:
same question here, do we want the helper function to throw an error? Or just log a warning/error

Reply:
THROW!